mismi-cli/main/s3.hs

```diff
@@ -181,11 +197,13 @@ text' = pack <$> (strArgument $
      metavar "STRING"
   <> help "Data to write to S3 address.")
 
-writeMode' :: Parser WriteMode
-writeMode' =
-  flag Fail Overwrite $
-       long "overwrite"
-    <> help "WriteMode"
+writeMode' :: Force -> Parser WriteMode
+writeMode' f =
+  case f of
+    Force ->
+      pure Overwrite
+    NotForce ->
+      flag Fail Overwrite $ long "overwrite" <> help "WriteMode"
```

Comment:
does this mean if you have `AWS_FORCE` set to `true`, you will get an error if you add `--overwrite`? sorry, not familiar enough to know